src/components/stepper/stepper.e2e.ts

```diff
@@ -602,5 +602,10 @@ describe("calcite-stepper", () => {
 
     const stepper1Number = await page.find("calcite-stepper-item[id='step-one'] >>> .stepper-item-number");
     expect(stepper1Number.textContent).toBe("1.");
+
+    stepper2.setProperty("numberingSystem", "thai");
+    await page.waitForChanges();
+    const stepper2Number = await page.find("calcite-stepper-item[id='step-two'] >>> .stepper-item-number");
+    expect(stepper2Number.textContent).toBe("๑.");
```

Comment:
Is it possible to avoid hardcoding the value that are non `latn` numbers so that we don't  need to change this if the localization API updates the string.

Reply:
Yeah I was going to use the `numerals` in `date-picker`'s Thai nls data but they are Latin:
https://github.com/Esri/calcite-components/blob/master/src/components/date-picker/assets/date-picker/nls/th.json#L13

Is there somewhere else that we put that data now after adding translations?